config/locales/fil/devise.fil.yml

```diff
@@ -0,0 +1,59 @@
+# Mga karagdagang mga pagsasalin ng salita ay makikita sa https://github.com/plataformatec/devise/wiki/I18n
+
+fil:
+  magbalangkas:
```

Comment:
`magbalangkas:` should be -> `devise:`

(We do not need to translate to the left <-- of the colon `:`)

Reply:
We only need to translate to the right --> of the colon `:`